### DISCUSSION TITLE
How do you bind a collection to a StackLayout?

### DISCUSSION THREAD

**Discussion by Sarund9** [picoe/Eto#1934] [Q&A]
How would I go about making this StackLayout display myCollection

``` c#
class ProgramWindow  {}

class MainWindow : Form
{
    HashSet<ProgramWindow> myCollection = new HashSet<ProgramWindow>();
    StackLayout tabs;

    void InitializeGUI() 
    {
        StackLayout layout = new StackLayout
        {
            // How to do Bindings
        };
        Content = layout;
    }
}
```

Note: i want to do a Direct binding

**Comment by cwensley** [comment]
Since it's a HashSet, which doesn't have any change notifications (e.g. `INotifyCollectionChanged`), you can simply just do a loop over the collection and add items to the `layout.Items` property, such as a `Label` or other control(s) to display whatever `ProgramWindow` is.
